pennylane_lightning/core/_serialize.py

```diff
@@ -307,11 +308,14 @@ def serialize_ops(
             for single_op in op_list:
                 name = single_op.name
                 names.append(name)
-
-                if not hasattr(self.sv_type, name):
+                # QubitUnitary is a special case, it has a parameter which is not differentiable.
+                # We thus pass a dummy 0.0 parameter which will not be referenced
+                if isinstance(single_op, QubitUnitary):
+                    params.append([0.0])
+                    mats.append(matrix(single_op))
```

Comment:
It looks like we have a coverage problem here, but the CIs are not showing it.

Reply:
It was not covered by `pennylane_lightning/core/_serialize.py`, but by other tests. Maybe that's why. In any case, I'll change the branching condition to use `name` and hit it with `pennylane_lightning/core/_serialize.py`. Thanks.